### ISSUE TITLE
Community: discord link dead

### ISSUE THREAD

**Issue by pintman** [pewpew-game/pewpew#23]
The link to the discord community seems to be dead.

https://github.com/pypewpew/pewpew/blob/master/docs/community.rst

**Comment by deshipu** [comment]
It works for me...

**Comment by deshipu** [comment]
Perhaps it only works when you are already in the Adafruit server, I should probably also add a link to https://adafru.it/discord in there.

**Comment by pintman** [comment]
Correct. After joining the adafruit server, I will be redirected to the correct channel. I would be helpful to have this in the documentation.
